examples/dem/3d-bouncing-particle/Bisection.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+def Young_effectif(Poisson_wall, Young_wall, Poisson_particule, Young_particule):
```

Comment:
Please rewrite the French word in English :).
Particule -> Particle
effectif - > effective, etc.